plc4j/drivers/ctrlx/src/main/java/org/apache/plc4x/java/ctrlx/readwrite/connection/CtrlXConnection.java

```diff
@@ -90,6 +93,17 @@ public void close() throws Exception {
         executorService.shutdown();
     }
 
+    @Override
+    public Optional<PlcTag> parseTagAddress(String tagAddress) {
+        PlcTag plcTag;
+        try {
+            plcTag = controlXTagHandler.parseTag(tagAddress);
+        } catch (Exception e) {
```

Comment:
the Exception should be at least logged at a debug level or if it never matters the variable should be renamed to `ignore`

Reply:
This is no real driver yet ... it's a skeleton, which I started working on, but then got all time stolen to continue working on it :-/